packages/core/src/baseclient.ts

```diff
@@ -413,6 +414,12 @@ export abstract class BaseClient<O extends ClientOptions> implements Client<O> {
   /** @inheritdoc */
   public on(hook: 'otelSpanEnd', callback: (otelSpan: unknown, mutableOptions: { drop: boolean }) => void): void;
 
+  /** @inheritdoc */
+  public on(
+    hook: 'afterPrepareFeedback',
```

Comment:
This generally looks good to me, but I'd prefer a different name (this sounds a bit implementation specific 🤔 ) what do you think about `'beforeSendFeedback'`, or something like this?

Reply:
👍 updated to use `beforeSendFeedback`

Reply:
Oops, too many branches